1033-textarea-design-guidance.md

```diff
@@ -0,0 +1,22 @@
+---
+category: component design guidance
+state: draft
+created: 2024-10-18
+updated: 2024-10-21
+---
+
+# Textarea Design Guidance
+
+[Textarea](https://clarity.design/documentation/textarea) component enable the user to input a single sentence or less of text.
+
+## Guidance
+
+- **Must** use Textarea when the expected user input may be more than a single line. If only a single line is expected, use [Input](https://clarity.design/documentation/input) instead.
+- **Should** provide a clear and concise label for context.
+- Placeholders **should** NOT be a substitute for a proper label element. It may only be used to show an example of the type of data that should be entered into a form. It **should** NOT be used in place of helper text either.
+- [States supported for Textarea](https://clarity.design/documentation/textarea#states).
+- Textarea is available in vertical, horizontal and compact layouts. [Layouts supported for Textarea](https://clarity.design/documentation/textarea#layouts)
```

Comment:
These two lines are not guidance. This is just documentation of available features. If we don't have guidance for when to use a specific state or layout, it shouldn't be here.

Reply:
Removed